src/devtools/client/debugger/src/components/TestInfo/ContextMenu.module.css

```diff
@@ -0,0 +1,41 @@
+.ContextMenu {
+  position: absolute;
+  flex-direction: column;
+  display: flex;
+  border-radius: 0.5em;
```

Comment:
Verifying we're using `em` intentionally throughout this since it'll scale in potentially unexpected ways with the font size change.

Reply:
It wasn't intentional, since I copied it from [here](https://github.com/replayio/devtools/blob/main/packages/bvaughn-architecture-demo/components/console/ContextMenu.module.css).

My instinct is to just change it to `rem` — thoughts?